src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/ClassUseWriter.java

```diff
@@ -230,24 +230,24 @@ protected void generateClassUseFile() throws DocFileIOException {
     /**
      * Add the class use documentation.
```

Comment:
I have a pair of classes I use in small utilities:
1. DeclScanner ... find accessible declarations in a `CompilationUnitTree`
2. (subtype of) DocTreeScanner ... analyze doc comments on those accessible declarations.
Whilst I know you are not a big fan of doc comments on internal API, we could still use them to find doc comments whose first sentence does not begin with 3rd person declarative form (the ...s form) of a verb.